host_ip: "auto"
host_udp_port_data: 55115
host_udp_port_control: 55116
sensor_ip: "192.168.0.10"
prefix: laser_1
angle_start: deg(-137.4)
angle_end: deg(137.4)
resolution: deg(0.2)
intensities: true
fragmented_scans: false